tests/cupyx_tests/scipy_tests/signal_tests/test_bsplines.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+
+from cupy import testing
+
+import cupyx.scipy.signal  # NOQA
+
+try:
+    import scipy.signal  # NOQA
+except ImportError:
+    pass
+
+
+@testing.parameterize(*testing.product({
+    'input': [(256, 256), (4, 512), (512, 3)],
+    'hrow': [1, 3, 4],
+    'hcol': [1, 3, 4],
+    'dtype': ['int32', 'float32', 'float64'],
```

Comment:
nit: Would you use data types instead of string literals here?
```suggestion
    'dtype': [cupy.int32, cupy.float32, cupy.float64],
```